<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>한국투자증권 시스템트레이딩 </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="accountTableView">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>831</width>
      <height>271</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌 정보</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>매수 조건</string>
    </property>
    <widget class="QSpinBox" name="buyRSIValueSpinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MACD가 Signal Line 을</string>
     </property>
    </widget>
    <widget class="QComboBox" name="buyMACDTypeComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>상향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>하향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이상</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이하</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="buyRSITypeComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>상향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>하향돌파</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>RSI 값이</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>매수당 금액 : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="buyAmountLineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10000</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>매도 조건</string>
    </property>
    <widget class="QSpinBox" name="sellRSIValueSpinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MACD가 Signal Line 을</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sellMACDTypeComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>상향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>하향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이상</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이하</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="sellRSITypeComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>상향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>하향돌파</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>47</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RSI 값이</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="watchListTableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>1101</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Watch List</string>
    </property>
   </widget>
   <widget class="QLabel" name="domesticCurrentBalanceLabel">
    <property name="geometry">
     <rect>
      <x>859</x>
      <y>6</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>현재평가잔고</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
